samples/create_thumbnail_from_mp4_video/GenerateThumbnailsPipeline.h

```diff
@@ -0,0 +1,30 @@
+#include <PipeLine.h>
+#include <ValveModule.h>
+#include "Mp4ReaderSource.h"
+#include "H264Decoder.h"
+#include "ColorConversionXForm.h"
+#include "CudaMemCopy.h"
+#include "FileWriterModule.h"
+#include "JPEGEncoderNVJPEG.h"
+
+
+
+class GenerateThumbnailsPipeline
+{
+public:
+    GenerateThumbnailsPipeline(); 
+    bool setUpPipeLine(const std::string &videoPath,const std::string &outFolderPath);
+    bool startPipeLine();
+    bool stopPipeLine();
+
+private:
+    PipeLine pipeLine;
```

Comment:
rename these variables to have this format: mValve, mMp4Reader, mPipeline and so on. m stands for class member variable